packages/amplication-design-system/src/components/UserAvatar/UserAvatar.scss

```diff
@@ -16,7 +16,7 @@
     text-align: center;
     width: var(--small-menu-item-width);
     color: var(--static-white);
-    font-size: var(--title2-font-size);
+    font-size: var(--text-field-label-font-size);
```

Comment:
You made a change to a design system. Did you test the impact on all other places where we use this component? Is this aligned with the design-system requirements?

Reply:
Yes, it is aligned with the design, and yes I tested the impact.

Reply:
@alexbass86 we saw together that the font needs to be 10px, right?